### PR TITLE
Stabilizations and improvements

### PR DESCRIPTION
**Pull request by sutaakar**
Using test method as a folder name simplifies searching for instance logs relevant to the test.

### CODE REVIEW

File: test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/scaling/KieServerWebSocketScalingIntegrationTest.java [hunk 1]
```diff
@@ -78,6 +80,9 @@
     private static final String WEBSOCKET_CONNECTION = "Connection to Kie Controller over Web Socket is now open";
     private static final String STARTED_CONTAINER = "Container cont-id (for release id org.kie.server.testing:definition-project-snapshot:1.0.0-SNAPSHOT) successfully started";
 
+    @Rule
+    public TestName testName= new TestName();
```
Comment (jakubschwan) [inline test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/scaling/KieServerWebSocketScalingIntegrationTest.java]: missing space before "="

File: test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/AbstractCloudIntegrationTest.java [hunk 2]
```diff
@@ -86,9 +87,13 @@
     protected final GitProvider gitProvider = gitProviderService.createGitProvider();
     protected T deploymentScenario;
 
+    @Rule
+    public TestName testName= new TestName();
```
Comment (jakubschwan) [inline test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/AbstractCloudIntegrationTest.java]: missing space before "="

File: test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/scaling/KieServerWebSocketScalingIntegrationTest.java (hunk 1 above)
Comment (sutaakar) [reply test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/scaling/KieServerWebSocketScalingIntegrationTest.java]: Done

File: test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/AbstractCloudIntegrationTest.java (hunk 2 above)
Comment (sutaakar) [reply test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/AbstractCloudIntegrationTest.java]: Done

**Pull request merged**